### PR TITLE
🐶 🍲

### PR DESCRIPTION
**Pull request by hzoo**

### CODE REVIEW

File: Gulpfile.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ var watch   = require("gulp-watch");
 var gutil   = require("gulp-util");
 var gulp    = require("gulp");
 var path    = require("path");
+var babelConfig = require("./.babelrc");
```
Comment (loganfsmyth) [inline Gulpfile.js]: Shouldn't it pick up the config automatically?

Comment (hzoo) [inline Gulpfile.js]: It was failing, not sure why so did this for now

File: .babelrc.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  "comments": false,
```
Comment (xtuc) [inline .babelrc.js]: Do we want to keep quotes on keys?

Comment (hzoo) [inline .babelrc.js]: We don't have to, I just copy pasted the config over!

File: scripts/babel-register.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 "use strict";
 
-const babel = require("../package.json").babel;
-const register = require("babel-register");
+const babel = require("../.babelrc");
+const register = require("babel-register").default;
 const path = require("path");
 
 if (babel.plugins) {
```
Comment (loganfsmyth) [inline scripts/babel-register.js]: Do we need this logic anymore?

File: .babelrc.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+let envOpts = {
+  loose: true
+};
+
+module.exports = {
+  comments: false,
+  presets: [
+    [
+      require("babel-preset-env"), envOpts
```
Comment (loganfsmyth) [inline .babelrc.js]: I think the `require` calls here are still optional, not sure if you had a specific motivation for them.

File: scripts/babel-register.js (hunk 3 above)
Comment (hzoo) [inline scripts/babel-register.js]: not sure why it was added before I guess not?

**Pull request merged**